components/Chat/Chat.vue

```diff
@@ -504,16 +461,18 @@
 
         console.log( `New alert:`, m );
 
+        // TODO: factor out these checks
         if ( m.channel
-          && m.channel.toLowerCase() !== this.page.toLowerCase() ) {
+          && this.$utils.normalizedCompare( m.channel, this.page ) ) {
           return;
         }
 
         // If alert is to us, make sound and read it
-        if ( m.channel.toLowerCase() === this.username.toLowerCase() ) {
+        if ( this.$utils.normalizedCompare( m.channel, this.username ) ) {
           // Play ping
           this.sound.play().then();
 
+          // TODO: getUseTtsAlerts has no UI, I think
```

Comment:
It does on the dev site
![image](https://user-images.githubusercontent.com/45262335/87421192-67c04000-c58b-11ea-965d-77c5ca2930d1.png)